model-server/src/main/kotlin/org/modelix/model/server/CmdLineArgs.kt

```diff
@@ -40,6 +40,9 @@ internal class CmdLineArgs {
     )
     var schemaInit = false
 
+    @Parameter(names = ["-useroleids"], description = "Use ids for roles instead of simpleNames", converter = BooleanConverter::class)
```

Comment:
```suggestion
    @Parameter(names = ["-useroleids"], description = "Use IDs for roles instead of simpleNames", converter = BooleanConverter::class)
```